releasing/plugin-commands-deploy/src/deploy.ts

```diff
@@ -72,6 +74,16 @@ export async function handler (
   const deployedDir = selectedDirs[0]
   const deployDirParam = params[0]
   const deployDir = path.isAbsolute(deployDirParam) ? deployDirParam : path.join(opts.dir, deployDirParam)
+
+  const isDeployDirEmpty = await isPathEmpty(deployDir)
```

Comment:
There is no benefit from making this function async. Nothing will run in parallel with this action, so it will be faster to make it sync. Please rewrite isEmptyDir to be sync